src/electrai/entrypoints/train.py

```diff
@@ -11,10 +11,20 @@
 from src.electrai.dataloader.registry import get_data
 from src.electrai.lightning import LightningGenerator
 from torch.utils.data import DataLoader
+from torch.utils.data._utils.collate import default_collate
```

Comment:
What's the motivation for importing `default_collate` from `torch.utils.data._utils.collate` rather than `torch.utils.data`?